### DISCUSSION TITLE
Is the website currently updated?, i uploaded a translation but i cant see it on the website.

### DISCUSSION THREAD

**Discussion by d4h0y0sr** [DivinumOfficium/divinum-officium#3437] [Q&A]
Hello, I uploaded a translation yesterday but i dont see it on the website, i know it works because i test it locally but i dont know if the website takes sometime to sync with the github changes.

Thanks

**Comment by benyanke** [comment]
Hello, thank you for your contribution to Divinum Officium! I’m the infrastructure admin. Deployments are actively something I’m working on improving here. It’s currently supposed to deploy within a few hours but it sometimes breaks - I’m working on a solution in the meantime to make it more robust and faster. In the meantime, I’ll go get things fixed. Thanks for letting us know!
